packages/utils/src/internals/robots.ts

```diff
@@ -83,7 +83,7 @@ export class RobotsFile {
      * @param [userAgent] relevant user agent, default to `*`
      */
     isAllowed(url: string, userAgent = '*'): boolean {
-        return this.robots.isAllowed(url, userAgent) ?? false;
+        return this.robots.isAllowed(url, userAgent) ?? true;
```

Comment:
```suggestion
        return this.robots.isAllowed(url, userAgent) ?? true; // `undefined` means that there is no explicit rule for the requested URL - assume it's allowed
```

Reply:
If this is OK @Sajito, please commit the suggestion :slightly_smiling_face:

Reply:
Done